[
  {
    "message": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "level": "FATAL",
    "type": "PLAYGROUND",
    "test": "ast-sqlite-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "log": "/tmp/tmp.CuuGYuXxoi"
  },
  {
    "message": "Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/ultimate-facebook-fan-box/plugin.php:214",
    "level": "FATAL",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:54:21 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/ultimate-facebook-fan-box/plugin.php:214\nStack trace:\n#0 /wordpress/wp-admin/includes/plugin.php(2387): include_once()\n#1 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('ultimate-facebo...')\n#2 /internal/eval.php(20): activate_plugin('ultimate-facebo...')\n#3 {main}\n  thrown in /wordpress/wp-content/plugins/ultimate-facebook-fan-box/plugin.php on line 214",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/ultimate-facebook-fan-box/plugin.php:214",
    "level": "FATAL",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:54:28 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/ultimate-facebook-fan-box/plugin.php:214\nStack trace:\n#0 /wordpress/wp-admin/includes/plugin.php(2387): include_once()\n#1 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('ultimate-facebo...')\n#2 /internal/eval.php(20): activate_plugin('ultimate-facebo...')\n#3 {main}\n  thrown in /wordpress/wp-content/plugins/ultimate-facebook-fan-box/plugin.php on line 214",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/ultimate-facebook-fan-box/plugin.php:214",
    "level": "FATAL",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:54:36 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/ultimate-facebook-fan-box/plugin.php:214\nStack trace:\n#0 /wordpress/wp-admin/includes/plugin.php(2387): include_once()\n#1 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('ultimate-facebo...')\n#2 /internal/eval.php(20): activate_plugin('ultimate-facebo...')\n#3 {main}\n  thrown in /wordpress/wp-content/plugins/ultimate-facebook-fan-box/plugin.php on line 214",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_VST_registros.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:55:23 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SELECT * FROM wp_VST_registros LIMIT 1</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: SELECT * FROM wp_VST_registros LIMIT 1 | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_VST_registros.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/visitors-app/includes/database.php(8): WP_SQLite_DB->query('SELECT * FROM w...')\n#2 /wordpress/wp-content/plugins/visitors-app/includes/activate_deactivate.php(9): VST_create_table_records()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): VST_activate(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_visito...', false)\n#7 /internal/eval.php(20): activate_plugin('visitors-app/Vi...')\n#8 {main}</pre>\n for query SELECT * FROM wp_VST_registros LIMIT 1 made by activate_plugin, do_action('activate_visitors-app/Visitors.php'), WP_Hook->do_action, WP_Hook->apply_filters, VST_activate, VST_create_table_records, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 near \"PRIMARY\": syntax error.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:55:23 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>\r\n\t\t\t\t\tALTER TABLE `wp_VST_registros` CHANGE `id` `id` INT(11) NOT NULL AUTO_INCREMENT, add PRIMARY KEY (`id`);\r\n\t\t\t\t</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_VST_registros, id, int(11)</li>\n<li>Executing: DROP TRIGGER IF EXISTS &quot;__wp_VST_registros_id_on_update__&quot; | (no parameters)</li>\n<li>Executing: SELECT sql FROM sqlite_master WHERE type=&quot;table&quot; AND name=:table | (no parameters)</li>\n<li>Executing: SELECT * FROM pragma_index_list(&quot;wp_VST_registros&quot;) as l; | (no parameters)</li>\n<li>Executing: CREATE TABLE `_tmp__wp_VST_registros_27827631` as SELECT * FROM `wp_VST_registros` | (no parameters)</li>\n<li>Executing: DROP TABLE `wp_VST_registros` | (no parameters)</li>\n<li>Executing: CREATE TABLE &quot;wp_VST_registros&quot; (\n       &quot;id&quot; integer PRIMARY KEY AUTOINCREMENT NOT NULL ON CONFLICT REPLACE DEFAULT 0,\n&quot;patch&quot; text DEFAULT NULL COLLATE NOCASE,\n&quot;datetime&quot; text DEFAULT NULL COLLATE NOCASE,\n&quot;useragent&quot; text DEFAULT NULL COLLATE NOCASE,\n&quot;ip&quot; text DEFAULT NULL COLLATE NOCASE)\n | (no parameters)</li>\n<li>Executing: INSERT INTO wp_VST_registros SELECT * FROM _tmp__wp_VST_registros_27827631 | (no parameters)</li>\n<li>Executing: DROP TABLE `_tmp__wp_VST_registros_27827631` | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_VST_registros, wp_VST_registros__(, PRIMARY</li>\n<li>Executing: CREATE PRIMARY KEY &quot;wp_VST_registros__(&quot; ON &quot;wp_VST_registros&quot; ();\r\n\t\t\t\t | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 near \"PRIMARY\": syntax error.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/visitors-app/includes/database.php(19): WP_SQLite_DB->query('\\r\\n\\t\\t\\t\\t\\tALTER TA...')\n#2 /wordpress/wp-content/plugins/visitors-app/includes/activate_deactivate.php(9): VST_create_table_records()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): VST_activate(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_visito...', false)\n#7 /internal/eval.php(20): activate_plugin('visitors-app/Vi...')\n#8 {main}</pre>\n for query \r\n\t\t\t\t\tALTER TABLE `wp_VST_registros` CHANGE `id` `id` INT(11) NOT NULL AUTO_INCREMENT, add PRIMARY KEY (`id`);\r\n\t\t\t\t made by activate_plugin, do_action('activate_visitors-app/Visitors.php'), WP_Hook->do_action, WP_Hook->apply_filters, VST_activate, VST_create_table_records, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_VST_registros.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:55:32 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SELECT * FROM wp_VST_registros LIMIT 1</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: SELECT * FROM wp_VST_registros LIMIT 1 | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_VST_registros.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/visitors-app/includes/database.php(8): WP_SQLite_DB->query('SELECT * FROM w...')\n#2 /wordpress/wp-content/plugins/visitors-app/includes/activate_deactivate.php(9): VST_create_table_records()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): VST_activate(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_visito...', false)\n#7 /internal/eval.php(20): activate_plugin('visitors-app/Vi...')\n#8 {main}</pre>\n for query SELECT * FROM wp_VST_registros LIMIT 1 made by activate_plugin, do_action('activate_visitors-app/Visitors.php'), WP_Hook->do_action, WP_Hook->apply_filters, VST_activate, VST_create_table_records, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 near \"PRIMARY\": syntax error.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:55:32 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>\r\n\t\t\t\t\tALTER TABLE `wp_VST_registros` CHANGE `id` `id` INT(11) NOT NULL AUTO_INCREMENT, add PRIMARY KEY (`id`);\r\n\t\t\t\t</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_VST_registros, id, int(11)</li>\n<li>Executing: DROP TRIGGER IF EXISTS &quot;__wp_VST_registros_id_on_update__&quot; | (no parameters)</li>\n<li>Executing: SELECT sql FROM sqlite_master WHERE type=&quot;table&quot; AND name=:table | (no parameters)</li>\n<li>Executing: SELECT * FROM pragma_index_list(&quot;wp_VST_registros&quot;) as l; | (no parameters)</li>\n<li>Executing: CREATE TABLE `_tmp__wp_VST_registros_98557970` as SELECT * FROM `wp_VST_registros` | (no parameters)</li>\n<li>Executing: DROP TABLE `wp_VST_registros` | (no parameters)</li>\n<li>Executing: CREATE TABLE &quot;wp_VST_registros&quot; (\n       &quot;id&quot; integer PRIMARY KEY AUTOINCREMENT NOT NULL ON CONFLICT REPLACE DEFAULT 0,\n&quot;patch&quot; text DEFAULT NULL COLLATE NOCASE,\n&quot;datetime&quot; text DEFAULT NULL COLLATE NOCASE,\n&quot;useragent&quot; text DEFAULT NULL COLLATE NOCASE,\n&quot;ip&quot; text DEFAULT NULL COLLATE NOCASE)\n | (no parameters)</li>\n<li>Executing: INSERT INTO wp_VST_registros SELECT * FROM _tmp__wp_VST_registros_98557970 | (no parameters)</li>\n<li>Executing: DROP TABLE `_tmp__wp_VST_registros_98557970` | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_VST_registros, wp_VST_registros__(, PRIMARY</li>\n<li>Executing: CREATE PRIMARY KEY &quot;wp_VST_registros__(&quot; ON &quot;wp_VST_registros&quot; ();\r\n\t\t\t\t | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 near \"PRIMARY\": syntax error.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/visitors-app/includes/database.php(19): WP_SQLite_DB->query('\\r\\n\\t\\t\\t\\t\\tALTER TA...')\n#2 /wordpress/wp-content/plugins/visitors-app/includes/activate_deactivate.php(9): VST_create_table_records()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): VST_activate(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_visito...', false)\n#7 /internal/eval.php(20): activate_plugin('visitors-app/Vi...')\n#8 {main}</pre>\n for query \r\n\t\t\t\t\tALTER TABLE `wp_VST_registros` CHANGE `id` `id` INT(11) NOT NULL AUTO_INCREMENT, add PRIMARY KEY (`id`);\r\n\t\t\t\t made by activate_plugin, do_action('activate_visitors-app/Visitors.php'), WP_Hook->do_action, WP_Hook->apply_filters, VST_activate, VST_create_table_records, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 near \",\": syntax error.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:55:53 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_vstrsnln_general` ADD UNIQUE ( `date_connection` ,`blog_id`)</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_vstrsnln_general, wp_vstrsnln_general__(, UNIQUE</li>\n<li>Executing: CREATE UNIQUE INDEX &quot;wp_vstrsnln_general__(&quot; ON &quot;wp_vstrsnln_general&quot; (  ,`blog_id`) | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 near \",\": syntax error.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(359): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(276): vstrsnln_install_base()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): vstrsnln_install(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_visito...', false)\n#7 /internal/eval.php(20): activate_plugin('visitors-online...')\n#8 {main}</pre>\n for query ALTER TABLE `wp_vstrsnln_general` ADD UNIQUE ( `date_connection` ,`blog_id`) made by activate_plugin, do_action('activate_visitors-online/visitors-online.php'), WP_Hook->do_action, WP_Hook->apply_filters, vstrsnln_install, vstrsnln_install_base, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 duplicate column name: country_id.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:55:53 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_vstrsnln_general` ADD `country_id` INT AFTER `blog_id`;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_vstrsnln_general, `COUNTRY_ID`, int</li>\n<li>Executing: ALTER TABLE wp_vstrsnln_general ADD `country_id` integer  ; | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 duplicate column name: country_id.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(365): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(276): vstrsnln_install_base()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): vstrsnln_install(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_visito...', false)\n#7 /internal/eval.php(20): activate_plugin('visitors-online...')\n#8 {main}</pre>\n for query ALTER TABLE `wp_vstrsnln_general` ADD `country_id` INT AFTER `blog_id`; made by activate_plugin, do_action('activate_visitors-online/visitors-online.php'), WP_Hook->do_action, WP_Hook->apply_filters, vstrsnln_install, vstrsnln_install_base, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such column: country.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:55:53 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SELECT DISTINCT `country` FROM `wp_vstrsnln_general`</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: SELECT DISTINCT `country` FROM `wp_vstrsnln_general` | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: country.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SELECT DISTINCT...')\n#2 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(367): wpdb->get_results('SELECT DISTINCT...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(276): vstrsnln_install_base()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): vstrsnln_install(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_visito...', false)\n#8 /internal/eval.php(20): activate_plugin('visitors-online...')\n#9 {main}</pre>\n for query SELECT DISTINCT `country` FROM `wp_vstrsnln_general` made by activate_plugin, do_action('activate_visitors-online/visitors-online.php'), WP_Hook->do_action, WP_Hook->apply_filters, vstrsnln_install, vstrsnln_install_base, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such column: \"`country`\".",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:55:53 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_vstrsnln_general` DROP COLUMN `country`;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ALTER TABLE wp_vstrsnln_general DROP COLUMN `country`; | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: \"`country`\".\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(393): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(276): vstrsnln_install_base()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): vstrsnln_install(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_visito...', false)\n#7 /internal/eval.php(20): activate_plugin('visitors-online...')\n#8 {main}</pre>\n for query ALTER TABLE `wp_vstrsnln_general` DROP COLUMN `country`; made by activate_plugin, do_action('activate_visitors-online/visitors-online.php'), WP_Hook->do_action, WP_Hook->apply_filters, vstrsnln_install, vstrsnln_install_base, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 near \",\": syntax error.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:56:03 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_vstrsnln_general` ADD UNIQUE ( `date_connection` ,`blog_id`)</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_vstrsnln_general, wp_vstrsnln_general__(, UNIQUE</li>\n<li>Executing: CREATE UNIQUE INDEX &quot;wp_vstrsnln_general__(&quot; ON &quot;wp_vstrsnln_general&quot; (  ,`blog_id`) | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 near \",\": syntax error.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(359): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(276): vstrsnln_install_base()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): vstrsnln_install(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_visito...', false)\n#7 /internal/eval.php(20): activate_plugin('visitors-online...')\n#8 {main}</pre>\n for query ALTER TABLE `wp_vstrsnln_general` ADD UNIQUE ( `date_connection` ,`blog_id`) made by activate_plugin, do_action('activate_visitors-online/visitors-online.php'), WP_Hook->do_action, WP_Hook->apply_filters, vstrsnln_install, vstrsnln_install_base, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 duplicate column name: country_id.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:56:03 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_vstrsnln_general` ADD `country_id` INT AFTER `blog_id`;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_vstrsnln_general, `COUNTRY_ID`, int</li>\n<li>Executing: ALTER TABLE wp_vstrsnln_general ADD `country_id` integer  ; | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 duplicate column name: country_id.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(365): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(276): vstrsnln_install_base()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): vstrsnln_install(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_visito...', false)\n#7 /internal/eval.php(20): activate_plugin('visitors-online...')\n#8 {main}</pre>\n for query ALTER TABLE `wp_vstrsnln_general` ADD `country_id` INT AFTER `blog_id`; made by activate_plugin, do_action('activate_visitors-online/visitors-online.php'), WP_Hook->do_action, WP_Hook->apply_filters, vstrsnln_install, vstrsnln_install_base, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such column: country.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:56:03 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SELECT DISTINCT `country` FROM `wp_vstrsnln_general`</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: SELECT DISTINCT `country` FROM `wp_vstrsnln_general` | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: country.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SELECT DISTINCT...')\n#2 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(367): wpdb->get_results('SELECT DISTINCT...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(276): vstrsnln_install_base()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): vstrsnln_install(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_visito...', false)\n#8 /internal/eval.php(20): activate_plugin('visitors-online...')\n#9 {main}</pre>\n for query SELECT DISTINCT `country` FROM `wp_vstrsnln_general` made by activate_plugin, do_action('activate_visitors-online/visitors-online.php'), WP_Hook->do_action, WP_Hook->apply_filters, vstrsnln_install, vstrsnln_install_base, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such column: \"`country`\".",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:56:03 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_vstrsnln_general` DROP COLUMN `country`;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ALTER TABLE wp_vstrsnln_general DROP COLUMN `country`; | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: \"`country`\".\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(393): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(276): vstrsnln_install_base()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): vstrsnln_install(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_visito...', false)\n#7 /internal/eval.php(20): activate_plugin('visitors-online...')\n#8 {main}</pre>\n for query ALTER TABLE `wp_vstrsnln_general` DROP COLUMN `country`; made by activate_plugin, do_action('activate_visitors-online/visitors-online.php'), WP_Hook->do_action, WP_Hook->apply_filters, vstrsnln_install, vstrsnln_install_base, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>visits_counter</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:56:15 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>visits_counter</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>visits_counter</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:56:24 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>visits_counter</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>visits_counter</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:56:33 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>visits_counter</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, function \"activate_vistrail\" not found or invalid function name in /wordpress/wp-includes/class-wp-hook.php:324",
    "level": "FATAL",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:56:43 UTC] PHP Fatal error:  Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, function \"activate_vistrail\" not found or invalid function name in /wordpress/wp-includes/class-wp-hook.php:324\nStack trace:\n#0 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#1 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#2 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_vistra...', false)\n#3 /internal/eval.php(20): activate_plugin('vistrail/vistra...')\n#4 {main}\n  thrown in /wordpress/wp-includes/class-wp-hook.php on line 324",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, function \"activate_vistrail\" not found or invalid function name in /wordpress/wp-includes/class-wp-hook.php:324",
    "level": "FATAL",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:56:50 UTC] PHP Fatal error:  Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, function \"activate_vistrail\" not found or invalid function name in /wordpress/wp-includes/class-wp-hook.php:324\nStack trace:\n#0 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#1 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#2 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_vistra...', false)\n#3 /internal/eval.php(20): activate_plugin('vistrail/vistra...')\n#4 {main}\n  thrown in /wordpress/wp-includes/class-wp-hook.php on line 324",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, function \"activate_vistrail\" not found or invalid function name in /wordpress/wp-includes/class-wp-hook.php:324",
    "level": "FATAL",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:56:58 UTC] PHP Fatal error:  Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, function \"activate_vistrail\" not found or invalid function name in /wordpress/wp-includes/class-wp-hook.php:324\nStack trace:\n#0 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#1 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#2 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_vistra...', false)\n#3 /internal/eval.php(20): activate_plugin('vistrail/vistra...')\n#4 {main}\n  thrown in /wordpress/wp-includes/class-wp-hook.php on line 324\n\n    at Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n    at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n  [cause]: Error: Error when executing the blueprint step #5 ({\"step\":\"installPlugin\",\"pluginData\":{\"resource\":\"wordpress.org/plugins\",\"slug\":\"visual-customizer-builder-lite\"},\"options\":{\"activate\":true}}) : Could not download \"https://downloads.wordpress.org/plugin/visual-customizer-builder-lite.latest-stable.zip\".\n  \t\t\t\tCheck if the URL is correct and the server is reachable.\n  \t\t\t\tIf it is reachable, the server might be blocking the request.\n  \t\t\t\tCheck the browser console and network tabs for more information.\n  \n  \t\t\t\t## Does the console show the error \"No 'Access-Control-Allow-Origin' header\"?\n  \n  \t\t\t\tThis means the server that hosts your file does not allow requests from other sites\n  \t\t\t\t(cross-origin requests, or CORS).\tYou need to move the asset to a server that allows\n  \t\t\t\tcross-origin file downloads. Learn more about CORS at\n  \t\t\t\thttps://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.\n  \n  \t\t\t\tIf your file is on GitHub, load it from \"raw.githubusercontent.com\".\n  \t\t\t\tHere's how to do that:\n  \n  \t\t\t\t1. Start with the original GitHub URL of the file. For example:\n  \t\t\t\thttps://github.com/username/repository/blob/branch/filename.\n  \t\t\t\t2. Replace \"github.com\" with \"raw.githubusercontent.com\".\n  \t\t\t\t3. Remove the \"/blob/\" part of the URL.\n  \n  \t\t\t\tThe resulting URL should look like this:\n  \t\t\t\thttps://raw.githubusercontent.com/username/repository/branch/filename\n  \n  \t\t\t\tError:\n  \t\t\t\tError: Could not download \"https://downloads.wordpress.org/plugin/visual-customizer-builder-lite.latest-stable.zip\"\n      at Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12449:19)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)\n      at async Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:227:48)\n      at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n      at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n    [cause]: Error: Could not download \"https://downloads.wordpress.org/plugin/visual-customizer-builder-lite.latest-stable.zip\".\n    \t\t\t\tCheck if the URL is correct and the server is reachable.\n    \t\t\t\tIf it is reachable, the server might be blocking the request.\n    \t\t\t\tCheck the browser console and network tabs for more information.\n    \n    \t\t\t\t## Does the console show the error \"No 'Access-Control-Allow-Origin' header\"?\n    \n    \t\t\t\tThis means the server that hosts your file does not allow requests from other sites\n    \t\t\t\t(cross-origin requests, or CORS).\tYou need to move the asset to a server that allows\n    \t\t\t\tcross-origin file downloads. Learn more about CORS at\n    \t\t\t\thttps://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.\n    \n    \t\t\t\tIf your file is on GitHub, load it from \"raw.githubusercontent.com\".\n    \t\t\t\tHere's how to do that:\n    \n    \t\t\t\t1. Start with the original GitHub URL of the file. For example:\n    \t\t\t\thttps://github.com/username/repository/blob/branch/filename.\n    \t\t\t\t2. Replace \"github.com\" with \"raw.githubusercontent.com\".\n    \t\t\t\t3. Remove the \"/blob/\" part of the URL.\n    \n    \t\t\t\tThe resulting URL should look like this:\n    \t\t\t\thttps://raw.githubusercontent.com/username/repository/branch/filename\n    \n    \t\t\t\tError:\n    \t\t\t\tError: Could not download \"https://downloads.wordpress.org/plugin/visual-customizer-builder-lite.latest-stable.zip\"\n        at Ct.resolve (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:1472:13)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async _.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@php-wasm/util/index.js:45:14)\n  }\n}\n\n",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "level": "FATAL",
    "type": "PLAYGROUND",
    "test": "asyncify-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "log": "/tmp/tmp.CQryDT6ub0"
  },
  {
    "message": "Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/ultimate-facebook-fan-box/plugin.php:214",
    "level": "FATAL",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:54:21 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/ultimate-facebook-fan-box/plugin.php:214\nStack trace:\n#0 /wordpress/wp-admin/includes/plugin.php(2387): include_once()\n#1 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('ultimate-facebo...')\n#2 /internal/eval.php(20): activate_plugin('ultimate-facebo...')\n#3 {main}\n  thrown in /wordpress/wp-content/plugins/ultimate-facebook-fan-box/plugin.php on line 214",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/ultimate-facebook-fan-box/plugin.php:214",
    "level": "FATAL",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:54:28 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/ultimate-facebook-fan-box/plugin.php:214\nStack trace:\n#0 /wordpress/wp-admin/includes/plugin.php(2387): include_once()\n#1 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('ultimate-facebo...')\n#2 /internal/eval.php(20): activate_plugin('ultimate-facebo...')\n#3 {main}\n  thrown in /wordpress/wp-content/plugins/ultimate-facebook-fan-box/plugin.php on line 214",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/ultimate-facebook-fan-box/plugin.php:214",
    "level": "FATAL",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:54:36 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/ultimate-facebook-fan-box/plugin.php:214\nStack trace:\n#0 /wordpress/wp-admin/includes/plugin.php(2387): include_once()\n#1 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('ultimate-facebo...')\n#2 /internal/eval.php(20): activate_plugin('ultimate-facebo...')\n#3 {main}\n  thrown in /wordpress/wp-content/plugins/ultimate-facebook-fan-box/plugin.php on line 214",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_VST_registros.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:55:23 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SELECT * FROM wp_VST_registros LIMIT 1</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: SELECT * FROM wp_VST_registros LIMIT 1 | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_VST_registros.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/visitors-app/includes/database.php(8): WP_SQLite_DB->query('SELECT * FROM w...')\n#2 /wordpress/wp-content/plugins/visitors-app/includes/activate_deactivate.php(9): VST_create_table_records()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): VST_activate(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_visito...', false)\n#7 /internal/eval.php(20): activate_plugin('visitors-app/Vi...')\n#8 {main}</pre>\n for query SELECT * FROM wp_VST_registros LIMIT 1 made by activate_plugin, do_action('activate_visitors-app/Visitors.php'), WP_Hook->do_action, WP_Hook->apply_filters, VST_activate, VST_create_table_records, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 near \"PRIMARY\": syntax error.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:55:23 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>\r\n\t\t\t\t\tALTER TABLE `wp_VST_registros` CHANGE `id` `id` INT(11) NOT NULL AUTO_INCREMENT, add PRIMARY KEY (`id`);\r\n\t\t\t\t</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_VST_registros, id, int(11)</li>\n<li>Executing: DROP TRIGGER IF EXISTS &quot;__wp_VST_registros_id_on_update__&quot; | (no parameters)</li>\n<li>Executing: SELECT sql FROM sqlite_master WHERE type=&quot;table&quot; AND name=:table | (no parameters)</li>\n<li>Executing: SELECT * FROM pragma_index_list(&quot;wp_VST_registros&quot;) as l; | (no parameters)</li>\n<li>Executing: CREATE TABLE `_tmp__wp_VST_registros_27827631` as SELECT * FROM `wp_VST_registros` | (no parameters)</li>\n<li>Executing: DROP TABLE `wp_VST_registros` | (no parameters)</li>\n<li>Executing: CREATE TABLE &quot;wp_VST_registros&quot; (\n       &quot;id&quot; integer PRIMARY KEY AUTOINCREMENT NOT NULL ON CONFLICT REPLACE DEFAULT 0,\n&quot;patch&quot; text DEFAULT NULL COLLATE NOCASE,\n&quot;datetime&quot; text DEFAULT NULL COLLATE NOCASE,\n&quot;useragent&quot; text DEFAULT NULL COLLATE NOCASE,\n&quot;ip&quot; text DEFAULT NULL COLLATE NOCASE)\n | (no parameters)</li>\n<li>Executing: INSERT INTO wp_VST_registros SELECT * FROM _tmp__wp_VST_registros_27827631 | (no parameters)</li>\n<li>Executing: DROP TABLE `_tmp__wp_VST_registros_27827631` | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_VST_registros, wp_VST_registros__(, PRIMARY</li>\n<li>Executing: CREATE PRIMARY KEY &quot;wp_VST_registros__(&quot; ON &quot;wp_VST_registros&quot; ();\r\n\t\t\t\t | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 near \"PRIMARY\": syntax error.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/visitors-app/includes/database.php(19): WP_SQLite_DB->query('\\r\\n\\t\\t\\t\\t\\tALTER TA...')\n#2 /wordpress/wp-content/plugins/visitors-app/includes/activate_deactivate.php(9): VST_create_table_records()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): VST_activate(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_visito...', false)\n#7 /internal/eval.php(20): activate_plugin('visitors-app/Vi...')\n#8 {main}</pre>\n for query \r\n\t\t\t\t\tALTER TABLE `wp_VST_registros` CHANGE `id` `id` INT(11) NOT NULL AUTO_INCREMENT, add PRIMARY KEY (`id`);\r\n\t\t\t\t made by activate_plugin, do_action('activate_visitors-app/Visitors.php'), WP_Hook->do_action, WP_Hook->apply_filters, VST_activate, VST_create_table_records, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_VST_registros.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:55:32 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SELECT * FROM wp_VST_registros LIMIT 1</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: SELECT * FROM wp_VST_registros LIMIT 1 | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_VST_registros.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/visitors-app/includes/database.php(8): WP_SQLite_DB->query('SELECT * FROM w...')\n#2 /wordpress/wp-content/plugins/visitors-app/includes/activate_deactivate.php(9): VST_create_table_records()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): VST_activate(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_visito...', false)\n#7 /internal/eval.php(20): activate_plugin('visitors-app/Vi...')\n#8 {main}</pre>\n for query SELECT * FROM wp_VST_registros LIMIT 1 made by activate_plugin, do_action('activate_visitors-app/Visitors.php'), WP_Hook->do_action, WP_Hook->apply_filters, VST_activate, VST_create_table_records, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 near \"PRIMARY\": syntax error.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:55:32 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>\r\n\t\t\t\t\tALTER TABLE `wp_VST_registros` CHANGE `id` `id` INT(11) NOT NULL AUTO_INCREMENT, add PRIMARY KEY (`id`);\r\n\t\t\t\t</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_VST_registros, id, int(11)</li>\n<li>Executing: DROP TRIGGER IF EXISTS &quot;__wp_VST_registros_id_on_update__&quot; | (no parameters)</li>\n<li>Executing: SELECT sql FROM sqlite_master WHERE type=&quot;table&quot; AND name=:table | (no parameters)</li>\n<li>Executing: SELECT * FROM pragma_index_list(&quot;wp_VST_registros&quot;) as l; | (no parameters)</li>\n<li>Executing: CREATE TABLE `_tmp__wp_VST_registros_98557970` as SELECT * FROM `wp_VST_registros` | (no parameters)</li>\n<li>Executing: DROP TABLE `wp_VST_registros` | (no parameters)</li>\n<li>Executing: CREATE TABLE &quot;wp_VST_registros&quot; (\n       &quot;id&quot; integer PRIMARY KEY AUTOINCREMENT NOT NULL ON CONFLICT REPLACE DEFAULT 0,\n&quot;patch&quot; text DEFAULT NULL COLLATE NOCASE,\n&quot;datetime&quot; text DEFAULT NULL COLLATE NOCASE,\n&quot;useragent&quot; text DEFAULT NULL COLLATE NOCASE,\n&quot;ip&quot; text DEFAULT NULL COLLATE NOCASE)\n | (no parameters)</li>\n<li>Executing: INSERT INTO wp_VST_registros SELECT * FROM _tmp__wp_VST_registros_98557970 | (no parameters)</li>\n<li>Executing: DROP TABLE `_tmp__wp_VST_registros_98557970` | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_VST_registros, wp_VST_registros__(, PRIMARY</li>\n<li>Executing: CREATE PRIMARY KEY &quot;wp_VST_registros__(&quot; ON &quot;wp_VST_registros&quot; ();\r\n\t\t\t\t | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 near \"PRIMARY\": syntax error.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/visitors-app/includes/database.php(19): WP_SQLite_DB->query('\\r\\n\\t\\t\\t\\t\\tALTER TA...')\n#2 /wordpress/wp-content/plugins/visitors-app/includes/activate_deactivate.php(9): VST_create_table_records()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): VST_activate(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_visito...', false)\n#7 /internal/eval.php(20): activate_plugin('visitors-app/Vi...')\n#8 {main}</pre>\n for query \r\n\t\t\t\t\tALTER TABLE `wp_VST_registros` CHANGE `id` `id` INT(11) NOT NULL AUTO_INCREMENT, add PRIMARY KEY (`id`);\r\n\t\t\t\t made by activate_plugin, do_action('activate_visitors-app/Visitors.php'), WP_Hook->do_action, WP_Hook->apply_filters, VST_activate, VST_create_table_records, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 near \",\": syntax error.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:55:53 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_vstrsnln_general` ADD UNIQUE ( `date_connection` ,`blog_id`)</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_vstrsnln_general, wp_vstrsnln_general__(, UNIQUE</li>\n<li>Executing: CREATE UNIQUE INDEX &quot;wp_vstrsnln_general__(&quot; ON &quot;wp_vstrsnln_general&quot; (  ,`blog_id`) | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 near \",\": syntax error.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(359): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(276): vstrsnln_install_base()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): vstrsnln_install(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_visito...', false)\n#7 /internal/eval.php(20): activate_plugin('visitors-online...')\n#8 {main}</pre>\n for query ALTER TABLE `wp_vstrsnln_general` ADD UNIQUE ( `date_connection` ,`blog_id`) made by activate_plugin, do_action('activate_visitors-online/visitors-online.php'), WP_Hook->do_action, WP_Hook->apply_filters, vstrsnln_install, vstrsnln_install_base, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 duplicate column name: country_id.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:55:53 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_vstrsnln_general` ADD `country_id` INT AFTER `blog_id`;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_vstrsnln_general, `COUNTRY_ID`, int</li>\n<li>Executing: ALTER TABLE wp_vstrsnln_general ADD `country_id` integer  ; | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 duplicate column name: country_id.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(365): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(276): vstrsnln_install_base()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): vstrsnln_install(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_visito...', false)\n#7 /internal/eval.php(20): activate_plugin('visitors-online...')\n#8 {main}</pre>\n for query ALTER TABLE `wp_vstrsnln_general` ADD `country_id` INT AFTER `blog_id`; made by activate_plugin, do_action('activate_visitors-online/visitors-online.php'), WP_Hook->do_action, WP_Hook->apply_filters, vstrsnln_install, vstrsnln_install_base, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such column: country.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:55:53 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SELECT DISTINCT `country` FROM `wp_vstrsnln_general`</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: SELECT DISTINCT `country` FROM `wp_vstrsnln_general` | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: country.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SELECT DISTINCT...')\n#2 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(367): wpdb->get_results('SELECT DISTINCT...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(276): vstrsnln_install_base()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): vstrsnln_install(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_visito...', false)\n#8 /internal/eval.php(20): activate_plugin('visitors-online...')\n#9 {main}</pre>\n for query SELECT DISTINCT `country` FROM `wp_vstrsnln_general` made by activate_plugin, do_action('activate_visitors-online/visitors-online.php'), WP_Hook->do_action, WP_Hook->apply_filters, vstrsnln_install, vstrsnln_install_base, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such column: \"`country`\".",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:55:53 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_vstrsnln_general` DROP COLUMN `country`;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ALTER TABLE wp_vstrsnln_general DROP COLUMN `country`; | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: \"`country`\".\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(393): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(276): vstrsnln_install_base()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): vstrsnln_install(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_visito...', false)\n#7 /internal/eval.php(20): activate_plugin('visitors-online...')\n#8 {main}</pre>\n for query ALTER TABLE `wp_vstrsnln_general` DROP COLUMN `country`; made by activate_plugin, do_action('activate_visitors-online/visitors-online.php'), WP_Hook->do_action, WP_Hook->apply_filters, vstrsnln_install, vstrsnln_install_base, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 near \",\": syntax error.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:56:03 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_vstrsnln_general` ADD UNIQUE ( `date_connection` ,`blog_id`)</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_vstrsnln_general, wp_vstrsnln_general__(, UNIQUE</li>\n<li>Executing: CREATE UNIQUE INDEX &quot;wp_vstrsnln_general__(&quot; ON &quot;wp_vstrsnln_general&quot; (  ,`blog_id`) | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 near \",\": syntax error.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(359): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(276): vstrsnln_install_base()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): vstrsnln_install(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_visito...', false)\n#7 /internal/eval.php(20): activate_plugin('visitors-online...')\n#8 {main}</pre>\n for query ALTER TABLE `wp_vstrsnln_general` ADD UNIQUE ( `date_connection` ,`blog_id`) made by activate_plugin, do_action('activate_visitors-online/visitors-online.php'), WP_Hook->do_action, WP_Hook->apply_filters, vstrsnln_install, vstrsnln_install_base, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 duplicate column name: country_id.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:56:03 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_vstrsnln_general` ADD `country_id` INT AFTER `blog_id`;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_vstrsnln_general, `COUNTRY_ID`, int</li>\n<li>Executing: ALTER TABLE wp_vstrsnln_general ADD `country_id` integer  ; | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 duplicate column name: country_id.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(365): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(276): vstrsnln_install_base()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): vstrsnln_install(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_visito...', false)\n#7 /internal/eval.php(20): activate_plugin('visitors-online...')\n#8 {main}</pre>\n for query ALTER TABLE `wp_vstrsnln_general` ADD `country_id` INT AFTER `blog_id`; made by activate_plugin, do_action('activate_visitors-online/visitors-online.php'), WP_Hook->do_action, WP_Hook->apply_filters, vstrsnln_install, vstrsnln_install_base, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such column: country.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:56:03 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SELECT DISTINCT `country` FROM `wp_vstrsnln_general`</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: SELECT DISTINCT `country` FROM `wp_vstrsnln_general` | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: country.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SELECT DISTINCT...')\n#2 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(367): wpdb->get_results('SELECT DISTINCT...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(276): vstrsnln_install_base()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): vstrsnln_install(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_visito...', false)\n#8 /internal/eval.php(20): activate_plugin('visitors-online...')\n#9 {main}</pre>\n for query SELECT DISTINCT `country` FROM `wp_vstrsnln_general` made by activate_plugin, do_action('activate_visitors-online/visitors-online.php'), WP_Hook->do_action, WP_Hook->apply_filters, vstrsnln_install, vstrsnln_install_base, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such column: \"`country`\".",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:56:03 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_vstrsnln_general` DROP COLUMN `country`;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ALTER TABLE wp_vstrsnln_general DROP COLUMN `country`; | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: \"`country`\".\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(393): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(276): vstrsnln_install_base()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): vstrsnln_install(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_visito...', false)\n#7 /internal/eval.php(20): activate_plugin('visitors-online...')\n#8 {main}</pre>\n for query ALTER TABLE `wp_vstrsnln_general` DROP COLUMN `country`; made by activate_plugin, do_action('activate_visitors-online/visitors-online.php'), WP_Hook->do_action, WP_Hook->apply_filters, vstrsnln_install, vstrsnln_install_base, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>visits_counter</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:56:15 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>visits_counter</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>visits_counter</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:56:24 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>visits_counter</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>visits_counter</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:56:33 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>visits_counter</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, function \"activate_vistrail\" not found or invalid function name in /wordpress/wp-includes/class-wp-hook.php:324",
    "level": "FATAL",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:56:43 UTC] PHP Fatal error:  Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, function \"activate_vistrail\" not found or invalid function name in /wordpress/wp-includes/class-wp-hook.php:324\nStack trace:\n#0 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#1 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#2 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_vistra...', false)\n#3 /internal/eval.php(20): activate_plugin('vistrail/vistra...')\n#4 {main}\n  thrown in /wordpress/wp-includes/class-wp-hook.php on line 324",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, function \"activate_vistrail\" not found or invalid function name in /wordpress/wp-includes/class-wp-hook.php:324",
    "level": "FATAL",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:56:50 UTC] PHP Fatal error:  Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, function \"activate_vistrail\" not found or invalid function name in /wordpress/wp-includes/class-wp-hook.php:324\nStack trace:\n#0 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#1 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#2 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_vistra...', false)\n#3 /internal/eval.php(20): activate_plugin('vistrail/vistra...')\n#4 {main}\n  thrown in /wordpress/wp-includes/class-wp-hook.php on line 324",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, function \"activate_vistrail\" not found or invalid function name in /wordpress/wp-includes/class-wp-hook.php:324",
    "level": "FATAL",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:56:58 UTC] PHP Fatal error:  Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, function \"activate_vistrail\" not found or invalid function name in /wordpress/wp-includes/class-wp-hook.php:324\nStack trace:\n#0 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#1 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#2 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_vistra...', false)\n#3 /internal/eval.php(20): activate_plugin('vistrail/vistra...')\n#4 {main}\n  thrown in /wordpress/wp-includes/class-wp-hook.php on line 324\n\n    at Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n    at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n  [cause]: Error: Error when executing the blueprint step #1 ({\"step\":\"installPlugin\",\"pluginData\":{\"resource\":\"wordpress.org/plugins\",\"slug\":\"visual-customizer-builder-lite\"},\"options\":{\"activate\":true}}) : Could not download \"https://downloads.wordpress.org/plugin/visual-customizer-builder-lite.latest-stable.zip\".\n  \t\t\t\tCheck if the URL is correct and the server is reachable.\n  \t\t\t\tIf it is reachable, the server might be blocking the request.\n  \t\t\t\tCheck the browser console and network tabs for more information.\n  \n  \t\t\t\t## Does the console show the error \"No 'Access-Control-Allow-Origin' header\"?\n  \n  \t\t\t\tThis means the server that hosts your file does not allow requests from other sites\n  \t\t\t\t(cross-origin requests, or CORS).\tYou need to move the asset to a server that allows\n  \t\t\t\tcross-origin file downloads. Learn more about CORS at\n  \t\t\t\thttps://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.\n  \n  \t\t\t\tIf your file is on GitHub, load it from \"raw.githubusercontent.com\".\n  \t\t\t\tHere's how to do that:\n  \n  \t\t\t\t1. Start with the original GitHub URL of the file. For example:\n  \t\t\t\thttps://github.com/username/repository/blob/branch/filename.\n  \t\t\t\t2. Replace \"github.com\" with \"raw.githubusercontent.com\".\n  \t\t\t\t3. Remove the \"/blob/\" part of the URL.\n  \n  \t\t\t\tThe resulting URL should look like this:\n  \t\t\t\thttps://raw.githubusercontent.com/username/repository/branch/filename\n  \n  \t\t\t\tError:\n  \t\t\t\tError: Could not download \"https://downloads.wordpress.org/plugin/visual-customizer-builder-lite.latest-stable.zip\"\n      at Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12449:19)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)\n      at async Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:227:48)\n      at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n      at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n    [cause]: Error: Could not download \"https://downloads.wordpress.org/plugin/visual-customizer-builder-lite.latest-stable.zip\".\n    \t\t\t\tCheck if the URL is correct and the server is reachable.\n    \t\t\t\tIf it is reachable, the server might be blocking the request.\n    \t\t\t\tCheck the browser console and network tabs for more information.\n    \n    \t\t\t\t## Does the console show the error \"No 'Access-Control-Allow-Origin' header\"?\n    \n    \t\t\t\tThis means the server that hosts your file does not allow requests from other sites\n    \t\t\t\t(cross-origin requests, or CORS).\tYou need to move the asset to a server that allows\n    \t\t\t\tcross-origin file downloads. Learn more about CORS at\n    \t\t\t\thttps://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.\n    \n    \t\t\t\tIf your file is on GitHub, load it from \"raw.githubusercontent.com\".\n    \t\t\t\tHere's how to do that:\n    \n    \t\t\t\t1. Start with the original GitHub URL of the file. For example:\n    \t\t\t\thttps://github.com/username/repository/blob/branch/filename.\n    \t\t\t\t2. Replace \"github.com\" with \"raw.githubusercontent.com\".\n    \t\t\t\t3. Remove the \"/blob/\" part of the URL.\n    \n    \t\t\t\tThe resulting URL should look like this:\n    \t\t\t\thttps://raw.githubusercontent.com/username/repository/branch/filename\n    \n    \t\t\t\tError:\n    \t\t\t\tError: Could not download \"https://downloads.wordpress.org/plugin/visual-customizer-builder-lite.latest-stable.zip\"\n        at Ct.resolve (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:1472:13)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async _.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@php-wasm/util/index.js:45:14)\n  }\n}\n\n",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "level": "FATAL",
    "type": "PLAYGROUND",
    "test": "jspi-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "log": "/tmp/tmp.ZqkJ0K91Mu"
  },
  {
    "message": "Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/ultimate-facebook-fan-box/plugin.php:214",
    "level": "FATAL",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:54:21 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/ultimate-facebook-fan-box/plugin.php:214\nStack trace:\n#0 /wordpress/wp-admin/includes/plugin.php(2387): include_once()\n#1 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('ultimate-facebo...')\n#2 /internal/eval.php(20): activate_plugin('ultimate-facebo...')\n#3 {main}\n  thrown in /wordpress/wp-content/plugins/ultimate-facebook-fan-box/plugin.php on line 214",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/ultimate-facebook-fan-box/plugin.php:214",
    "level": "FATAL",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:54:28 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/ultimate-facebook-fan-box/plugin.php:214\nStack trace:\n#0 /wordpress/wp-admin/includes/plugin.php(2387): include_once()\n#1 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('ultimate-facebo...')\n#2 /internal/eval.php(20): activate_plugin('ultimate-facebo...')\n#3 {main}\n  thrown in /wordpress/wp-content/plugins/ultimate-facebook-fan-box/plugin.php on line 214",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/ultimate-facebook-fan-box/plugin.php:214",
    "level": "FATAL",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:54:36 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/ultimate-facebook-fan-box/plugin.php:214\nStack trace:\n#0 /wordpress/wp-admin/includes/plugin.php(2387): include_once()\n#1 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('ultimate-facebo...')\n#2 /internal/eval.php(20): activate_plugin('ultimate-facebo...')\n#3 {main}\n  thrown in /wordpress/wp-content/plugins/ultimate-facebook-fan-box/plugin.php on line 214",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_VST_registros.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:55:23 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SELECT * FROM wp_VST_registros LIMIT 1</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: SELECT * FROM wp_VST_registros LIMIT 1 | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_VST_registros.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/visitors-app/includes/database.php(8): WP_SQLite_DB->query('SELECT * FROM w...')\n#2 /wordpress/wp-content/plugins/visitors-app/includes/activate_deactivate.php(9): VST_create_table_records()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): VST_activate(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_visito...', false)\n#7 /internal/eval.php(20): activate_plugin('visitors-app/Vi...')\n#8 {main}</pre>\n for query SELECT * FROM wp_VST_registros LIMIT 1 made by activate_plugin, do_action('activate_visitors-app/Visitors.php'), WP_Hook->do_action, WP_Hook->apply_filters, VST_activate, VST_create_table_records, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 near \"PRIMARY\": syntax error.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:55:23 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>\r\n\t\t\t\t\tALTER TABLE `wp_VST_registros` CHANGE `id` `id` INT(11) NOT NULL AUTO_INCREMENT, add PRIMARY KEY (`id`);\r\n\t\t\t\t</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_VST_registros, id, int(11)</li>\n<li>Executing: DROP TRIGGER IF EXISTS &quot;__wp_VST_registros_id_on_update__&quot; | (no parameters)</li>\n<li>Executing: SELECT sql FROM sqlite_master WHERE type=&quot;table&quot; AND name=:table | (no parameters)</li>\n<li>Executing: SELECT * FROM pragma_index_list(&quot;wp_VST_registros&quot;) as l; | (no parameters)</li>\n<li>Executing: CREATE TABLE `_tmp__wp_VST_registros_27827631` as SELECT * FROM `wp_VST_registros` | (no parameters)</li>\n<li>Executing: DROP TABLE `wp_VST_registros` | (no parameters)</li>\n<li>Executing: CREATE TABLE &quot;wp_VST_registros&quot; (\n       &quot;id&quot; integer PRIMARY KEY AUTOINCREMENT NOT NULL ON CONFLICT REPLACE DEFAULT 0,\n&quot;patch&quot; text DEFAULT NULL COLLATE NOCASE,\n&quot;datetime&quot; text DEFAULT NULL COLLATE NOCASE,\n&quot;useragent&quot; text DEFAULT NULL COLLATE NOCASE,\n&quot;ip&quot; text DEFAULT NULL COLLATE NOCASE)\n | (no parameters)</li>\n<li>Executing: INSERT INTO wp_VST_registros SELECT * FROM _tmp__wp_VST_registros_27827631 | (no parameters)</li>\n<li>Executing: DROP TABLE `_tmp__wp_VST_registros_27827631` | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_VST_registros, wp_VST_registros__(, PRIMARY</li>\n<li>Executing: CREATE PRIMARY KEY &quot;wp_VST_registros__(&quot; ON &quot;wp_VST_registros&quot; ();\r\n\t\t\t\t | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 near \"PRIMARY\": syntax error.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/visitors-app/includes/database.php(19): WP_SQLite_DB->query('\\r\\n\\t\\t\\t\\t\\tALTER TA...')\n#2 /wordpress/wp-content/plugins/visitors-app/includes/activate_deactivate.php(9): VST_create_table_records()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): VST_activate(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_visito...', false)\n#7 /internal/eval.php(20): activate_plugin('visitors-app/Vi...')\n#8 {main}</pre>\n for query \r\n\t\t\t\t\tALTER TABLE `wp_VST_registros` CHANGE `id` `id` INT(11) NOT NULL AUTO_INCREMENT, add PRIMARY KEY (`id`);\r\n\t\t\t\t made by activate_plugin, do_action('activate_visitors-app/Visitors.php'), WP_Hook->do_action, WP_Hook->apply_filters, VST_activate, VST_create_table_records, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_VST_registros.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:55:32 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SELECT * FROM wp_VST_registros LIMIT 1</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: SELECT * FROM wp_VST_registros LIMIT 1 | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_VST_registros.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/visitors-app/includes/database.php(8): WP_SQLite_DB->query('SELECT * FROM w...')\n#2 /wordpress/wp-content/plugins/visitors-app/includes/activate_deactivate.php(9): VST_create_table_records()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): VST_activate(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_visito...', false)\n#7 /internal/eval.php(20): activate_plugin('visitors-app/Vi...')\n#8 {main}</pre>\n for query SELECT * FROM wp_VST_registros LIMIT 1 made by activate_plugin, do_action('activate_visitors-app/Visitors.php'), WP_Hook->do_action, WP_Hook->apply_filters, VST_activate, VST_create_table_records, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 near \"PRIMARY\": syntax error.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:55:32 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>\r\n\t\t\t\t\tALTER TABLE `wp_VST_registros` CHANGE `id` `id` INT(11) NOT NULL AUTO_INCREMENT, add PRIMARY KEY (`id`);\r\n\t\t\t\t</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_VST_registros, id, int(11)</li>\n<li>Executing: DROP TRIGGER IF EXISTS &quot;__wp_VST_registros_id_on_update__&quot; | (no parameters)</li>\n<li>Executing: SELECT sql FROM sqlite_master WHERE type=&quot;table&quot; AND name=:table | (no parameters)</li>\n<li>Executing: SELECT * FROM pragma_index_list(&quot;wp_VST_registros&quot;) as l; | (no parameters)</li>\n<li>Executing: CREATE TABLE `_tmp__wp_VST_registros_98557970` as SELECT * FROM `wp_VST_registros` | (no parameters)</li>\n<li>Executing: DROP TABLE `wp_VST_registros` | (no parameters)</li>\n<li>Executing: CREATE TABLE &quot;wp_VST_registros&quot; (\n       &quot;id&quot; integer PRIMARY KEY AUTOINCREMENT NOT NULL ON CONFLICT REPLACE DEFAULT 0,\n&quot;patch&quot; text DEFAULT NULL COLLATE NOCASE,\n&quot;datetime&quot; text DEFAULT NULL COLLATE NOCASE,\n&quot;useragent&quot; text DEFAULT NULL COLLATE NOCASE,\n&quot;ip&quot; text DEFAULT NULL COLLATE NOCASE)\n | (no parameters)</li>\n<li>Executing: INSERT INTO wp_VST_registros SELECT * FROM _tmp__wp_VST_registros_98557970 | (no parameters)</li>\n<li>Executing: DROP TABLE `_tmp__wp_VST_registros_98557970` | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_VST_registros, wp_VST_registros__(, PRIMARY</li>\n<li>Executing: CREATE PRIMARY KEY &quot;wp_VST_registros__(&quot; ON &quot;wp_VST_registros&quot; ();\r\n\t\t\t\t | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 near \"PRIMARY\": syntax error.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/visitors-app/includes/database.php(19): WP_SQLite_DB->query('\\r\\n\\t\\t\\t\\t\\tALTER TA...')\n#2 /wordpress/wp-content/plugins/visitors-app/includes/activate_deactivate.php(9): VST_create_table_records()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): VST_activate(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_visito...', false)\n#7 /internal/eval.php(20): activate_plugin('visitors-app/Vi...')\n#8 {main}</pre>\n for query \r\n\t\t\t\t\tALTER TABLE `wp_VST_registros` CHANGE `id` `id` INT(11) NOT NULL AUTO_INCREMENT, add PRIMARY KEY (`id`);\r\n\t\t\t\t made by activate_plugin, do_action('activate_visitors-app/Visitors.php'), WP_Hook->do_action, WP_Hook->apply_filters, VST_activate, VST_create_table_records, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 near \",\": syntax error.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:55:53 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_vstrsnln_general` ADD UNIQUE ( `date_connection` ,`blog_id`)</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_vstrsnln_general, wp_vstrsnln_general__(, UNIQUE</li>\n<li>Executing: CREATE UNIQUE INDEX &quot;wp_vstrsnln_general__(&quot; ON &quot;wp_vstrsnln_general&quot; (  ,`blog_id`) | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 near \",\": syntax error.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(359): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(276): vstrsnln_install_base()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): vstrsnln_install(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_visito...', false)\n#7 /internal/eval.php(20): activate_plugin('visitors-online...')\n#8 {main}</pre>\n for query ALTER TABLE `wp_vstrsnln_general` ADD UNIQUE ( `date_connection` ,`blog_id`) made by activate_plugin, do_action('activate_visitors-online/visitors-online.php'), WP_Hook->do_action, WP_Hook->apply_filters, vstrsnln_install, vstrsnln_install_base, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 duplicate column name: country_id.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:55:53 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_vstrsnln_general` ADD `country_id` INT AFTER `blog_id`;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_vstrsnln_general, `COUNTRY_ID`, int</li>\n<li>Executing: ALTER TABLE wp_vstrsnln_general ADD `country_id` integer  ; | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 duplicate column name: country_id.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(365): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(276): vstrsnln_install_base()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): vstrsnln_install(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_visito...', false)\n#7 /internal/eval.php(20): activate_plugin('visitors-online...')\n#8 {main}</pre>\n for query ALTER TABLE `wp_vstrsnln_general` ADD `country_id` INT AFTER `blog_id`; made by activate_plugin, do_action('activate_visitors-online/visitors-online.php'), WP_Hook->do_action, WP_Hook->apply_filters, vstrsnln_install, vstrsnln_install_base, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such column: country.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:55:53 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SELECT DISTINCT `country` FROM `wp_vstrsnln_general`</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: SELECT DISTINCT `country` FROM `wp_vstrsnln_general` | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: country.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SELECT DISTINCT...')\n#2 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(367): wpdb->get_results('SELECT DISTINCT...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(276): vstrsnln_install_base()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): vstrsnln_install(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_visito...', false)\n#8 /internal/eval.php(20): activate_plugin('visitors-online...')\n#9 {main}</pre>\n for query SELECT DISTINCT `country` FROM `wp_vstrsnln_general` made by activate_plugin, do_action('activate_visitors-online/visitors-online.php'), WP_Hook->do_action, WP_Hook->apply_filters, vstrsnln_install, vstrsnln_install_base, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such column: \"`country`\".",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:55:53 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_vstrsnln_general` DROP COLUMN `country`;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ALTER TABLE wp_vstrsnln_general DROP COLUMN `country`; | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: \"`country`\".\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(393): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(276): vstrsnln_install_base()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): vstrsnln_install(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_visito...', false)\n#7 /internal/eval.php(20): activate_plugin('visitors-online...')\n#8 {main}</pre>\n for query ALTER TABLE `wp_vstrsnln_general` DROP COLUMN `country`; made by activate_plugin, do_action('activate_visitors-online/visitors-online.php'), WP_Hook->do_action, WP_Hook->apply_filters, vstrsnln_install, vstrsnln_install_base, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 near \",\": syntax error.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:56:03 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_vstrsnln_general` ADD UNIQUE ( `date_connection` ,`blog_id`)</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_vstrsnln_general, wp_vstrsnln_general__(, UNIQUE</li>\n<li>Executing: CREATE UNIQUE INDEX &quot;wp_vstrsnln_general__(&quot; ON &quot;wp_vstrsnln_general&quot; (  ,`blog_id`) | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 near \",\": syntax error.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(359): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(276): vstrsnln_install_base()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): vstrsnln_install(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_visito...', false)\n#7 /internal/eval.php(20): activate_plugin('visitors-online...')\n#8 {main}</pre>\n for query ALTER TABLE `wp_vstrsnln_general` ADD UNIQUE ( `date_connection` ,`blog_id`) made by activate_plugin, do_action('activate_visitors-online/visitors-online.php'), WP_Hook->do_action, WP_Hook->apply_filters, vstrsnln_install, vstrsnln_install_base, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 duplicate column name: country_id.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:56:03 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_vstrsnln_general` ADD `country_id` INT AFTER `blog_id`;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_vstrsnln_general, `COUNTRY_ID`, int</li>\n<li>Executing: ALTER TABLE wp_vstrsnln_general ADD `country_id` integer  ; | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 duplicate column name: country_id.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(365): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(276): vstrsnln_install_base()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): vstrsnln_install(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_visito...', false)\n#7 /internal/eval.php(20): activate_plugin('visitors-online...')\n#8 {main}</pre>\n for query ALTER TABLE `wp_vstrsnln_general` ADD `country_id` INT AFTER `blog_id`; made by activate_plugin, do_action('activate_visitors-online/visitors-online.php'), WP_Hook->do_action, WP_Hook->apply_filters, vstrsnln_install, vstrsnln_install_base, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such column: country.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:56:03 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SELECT DISTINCT `country` FROM `wp_vstrsnln_general`</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: SELECT DISTINCT `country` FROM `wp_vstrsnln_general` | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: country.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SELECT DISTINCT...')\n#2 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(367): wpdb->get_results('SELECT DISTINCT...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(276): vstrsnln_install_base()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): vstrsnln_install(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_visito...', false)\n#8 /internal/eval.php(20): activate_plugin('visitors-online...')\n#9 {main}</pre>\n for query SELECT DISTINCT `country` FROM `wp_vstrsnln_general` made by activate_plugin, do_action('activate_visitors-online/visitors-online.php'), WP_Hook->do_action, WP_Hook->apply_filters, vstrsnln_install, vstrsnln_install_base, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such column: \"`country`\".",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:56:03 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_vstrsnln_general` DROP COLUMN `country`;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ALTER TABLE wp_vstrsnln_general DROP COLUMN `country`; | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: \"`country`\".\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(393): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/visitors-online/visitors-online.php(276): vstrsnln_install_base()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): vstrsnln_install(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_visito...', false)\n#7 /internal/eval.php(20): activate_plugin('visitors-online...')\n#8 {main}</pre>\n for query ALTER TABLE `wp_vstrsnln_general` DROP COLUMN `country`; made by activate_plugin, do_action('activate_visitors-online/visitors-online.php'), WP_Hook->do_action, WP_Hook->apply_filters, vstrsnln_install, vstrsnln_install_base, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>visits_counter</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:56:15 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>visits_counter</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>visits_counter</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:56:24 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>visits_counter</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>visits_counter</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:56:33 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>visits_counter</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, function \"activate_vistrail\" not found or invalid function name in /wordpress/wp-includes/class-wp-hook.php:324",
    "level": "FATAL",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:56:43 UTC] PHP Fatal error:  Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, function \"activate_vistrail\" not found or invalid function name in /wordpress/wp-includes/class-wp-hook.php:324\nStack trace:\n#0 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#1 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#2 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_vistra...', false)\n#3 /internal/eval.php(20): activate_plugin('vistrail/vistra...')\n#4 {main}\n  thrown in /wordpress/wp-includes/class-wp-hook.php on line 324",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, function \"activate_vistrail\" not found or invalid function name in /wordpress/wp-includes/class-wp-hook.php:324",
    "level": "FATAL",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:56:50 UTC] PHP Fatal error:  Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, function \"activate_vistrail\" not found or invalid function name in /wordpress/wp-includes/class-wp-hook.php:324\nStack trace:\n#0 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#1 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#2 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_vistra...', false)\n#3 /internal/eval.php(20): activate_plugin('vistrail/vistra...')\n#4 {main}\n  thrown in /wordpress/wp-includes/class-wp-hook.php on line 324",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  },
  {
    "message": "Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, function \"activate_vistrail\" not found or invalid function name in /wordpress/wp-includes/class-wp-hook.php:324",
    "level": "FATAL",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "visual-customizer-builder-lite",
    "details": "[22-May-2025 10:56:58 UTC] PHP Fatal error:  Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, function \"activate_vistrail\" not found or invalid function name in /wordpress/wp-includes/class-wp-hook.php:324\nStack trace:\n#0 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#1 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#2 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_vistra...', false)\n#3 /internal/eval.php(20): activate_plugin('vistrail/vistra...')\n#4 {main}\n  thrown in /wordpress/wp-includes/class-wp-hook.php on line 324\n\n    at Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n    at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n  [cause]: Error: Error when executing the blueprint step #1 ({\"step\":\"installPlugin\",\"pluginData\":{\"resource\":\"wordpress.org/plugins\",\"slug\":\"visual-customizer-builder-lite\"},\"options\":{\"activate\":true}}) : Could not download \"https://downloads.wordpress.org/plugin/visual-customizer-builder-lite.latest-stable.zip\".\n  \t\t\t\tCheck if the URL is correct and the server is reachable.\n  \t\t\t\tIf it is reachable, the server might be blocking the request.\n  \t\t\t\tCheck the browser console and network tabs for more information.\n  \n  \t\t\t\t## Does the console show the error \"No 'Access-Control-Allow-Origin' header\"?\n  \n  \t\t\t\tThis means the server that hosts your file does not allow requests from other sites\n  \t\t\t\t(cross-origin requests, or CORS).\tYou need to move the asset to a server that allows\n  \t\t\t\tcross-origin file downloads. Learn more about CORS at\n  \t\t\t\thttps://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.\n  \n  \t\t\t\tIf your file is on GitHub, load it from \"raw.githubusercontent.com\".\n  \t\t\t\tHere's how to do that:\n  \n  \t\t\t\t1. Start with the original GitHub URL of the file. For example:\n  \t\t\t\thttps://github.com/username/repository/blob/branch/filename.\n  \t\t\t\t2. Replace \"github.com\" with \"raw.githubusercontent.com\".\n  \t\t\t\t3. Remove the \"/blob/\" part of the URL.\n  \n  \t\t\t\tThe resulting URL should look like this:\n  \t\t\t\thttps://raw.githubusercontent.com/username/repository/branch/filename\n  \n  \t\t\t\tError:\n  \t\t\t\tError: Could not download \"https://downloads.wordpress.org/plugin/visual-customizer-builder-lite.latest-stable.zip\"\n      at Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12449:19)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)\n      at async Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:227:48)\n      at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n      at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n    [cause]: Error: Could not download \"https://downloads.wordpress.org/plugin/visual-customizer-builder-lite.latest-stable.zip\".\n    \t\t\t\tCheck if the URL is correct and the server is reachable.\n    \t\t\t\tIf it is reachable, the server might be blocking the request.\n    \t\t\t\tCheck the browser console and network tabs for more information.\n    \n    \t\t\t\t## Does the console show the error \"No 'Access-Control-Allow-Origin' header\"?\n    \n    \t\t\t\tThis means the server that hosts your file does not allow requests from other sites\n    \t\t\t\t(cross-origin requests, or CORS).\tYou need to move the asset to a server that allows\n    \t\t\t\tcross-origin file downloads. Learn more about CORS at\n    \t\t\t\thttps://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.\n    \n    \t\t\t\tIf your file is on GitHub, load it from \"raw.githubusercontent.com\".\n    \t\t\t\tHere's how to do that:\n    \n    \t\t\t\t1. Start with the original GitHub URL of the file. For example:\n    \t\t\t\thttps://github.com/username/repository/blob/branch/filename.\n    \t\t\t\t2. Replace \"github.com\" with \"raw.githubusercontent.com\".\n    \t\t\t\t3. Remove the \"/blob/\" part of the URL.\n    \n    \t\t\t\tThe resulting URL should look like this:\n    \t\t\t\thttps://raw.githubusercontent.com/username/repository/branch/filename\n    \n    \t\t\t\tError:\n    \t\t\t\tError: Could not download \"https://downloads.wordpress.org/plugin/visual-customizer-builder-lite.latest-stable.zip\"\n        at Ct.resolve (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:1472:13)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async _.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@php-wasm/util/index.js:45:14)\n  }\n}\n\n",
    "log": "/logs/plugins/v/visual-customizer-builder-lite/error.json"
  }
]
